build.gradle

```diff
@@ -499,7 +533,9 @@ task allJavaDoc(type: Javadoc) {
 }
 
 
-task jenkins(dependsOn: ['replaceVersion', subprojects.build, 'binDistZip','binDistTar','srcDistZip','srcDistTar']) {}
+task jenkins(dependsOn: ['replaceVersion', subprojects.build, 'binDistZip','binDistTar','srcDistZip','srcDistTar']) {
+	isCIBuild = true;
```

Comment:
I actually use the "jenkins" task also locally as a "Pre-Commit-Check", as otherwise things like "rat-check" and "javadoc" are not run and so you potentially commit changes which break the build.

We can safely set the environment-variables as part of the CI builds via our DSL-based approach, see https://github.com/apache/poi/blob/trunk/jenkins/create_jobs.groovy#L264 for the related code-based CI configuration.

Reply:
Sorry for being slow: does `isCIBuild = true` do any harm?
I could remove that easily of course, but I found it handy when `jenkins` always simulated the CI build.

But yes, adding the Environment Variables and also setting the Gradle Switches at https://github.com/apache/poi/blob/584c8c059b6b317178df635b37244c6e393d6c96/jenkins/create_jobs.groovy#L372 should do.

Can someone please try? :-D